File: chap15/pro15/src/main/java/org/mklinkj/taojwp/common/CommonUtils.java
package org.mklinkj.taojwp.common;

import java.nio.file.Paths;
import java.util.Optional;
import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class CommonUtils {
  public static String ifNullToNullString(String string) {
    return String.valueOf(string);
  }

  public static String fileNameOnly(String fileName) {
    return Paths.get(fileName).getFileName().toString();
  }
}

File: chap19/pro19/src/main/java/org/mklinkj/taojwp/common/util/CommonUtils.kt
package org.mklinkj.taojwp.common.util

import java.nio.file.Paths

object CommonUtils {
    @JvmStatic
    fun ifNullToNullString(string: String?): String {
        return string ?: "null"
    }

    @JvmStatic
    fun fileNameOnly(fileName: String?): String {
        return Paths.get(fileName).fileName.toString()
    }
}
